src/js/components/ChromecastButton.js

```diff
@@ -51,8 +51,12 @@ ChromecastButton = {
     * @param el {DOMElement}
     * @see {@link http://docs.videojs.com/Button.html#buildCSSClass|Button#buildCSSClass}
     */
-   buildCSSClass: function() {
-      return 'vjs-chromecast-button ' + (this._isChromecastConnected ? 'vjs-chromecast-casting-state ' : '') +
+   buildCSSClass: function(isChromecastConnected) {
```

Comment:
Are these arguments compatible with video.js's original ones?

Reply:
Yes, it works essentially the same, except that now we cover an edge case when render was called before the event handlers

Reply:
It could be but I don't see much value in saving `this._isChromecastConnected`, as it can introduce an error that when `buildCSSClass` will be called it will act on outdated information.

Reply:
😑